addon/components/o-s-s/number-input.ts

```diff
@@ -76,4 +89,24 @@ export default class OSSNumberInput extends Component<OSSNumberInputArgs> {
   notifyChanges(): void {
     this.args.onChange?.(Number(this.localValue));
   }
+
+  private reachedTooltipHandler(): void {
+    if (Number(this.localValue) === this.args.min) {
+      if (!this.args.minReachedTooltip) return;
+      this.reachedTooltip = this.args.minReachedTooltip;
+      return;
+    }
+    if (Number(this.localValue) === this.args.max) {
+      if (!this.args.maxReachedTooltip) return;
+      this.reachedTooltip = this.args.maxReachedTooltip;
+      return;
+    }
+    this.reachedTooltip = null;
+    document.querySelector('.upf-tooltip')?.remove();
```

Comment:
maybe these can be getters as well and you might be able to duplicate the buttons in the template to prevent the "null" tooltip from displaying